src/edu/stanford/dlss/was/WasapiDownloaderSettings.java

```diff
@@ -5,46 +5,157 @@
 import java.io.IOException;
 import java.util.Properties;
 
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
 public class WasapiDownloaderSettings {
+  // a small inner class to represent exceptions in general when loading the config.  prevents callers from
+  // having to know too much detail about more granular exceptions that might get thrown when reading properties,
+  // parsing CLI args, etc.
+  class LoadSettingsException extends Exception {
```

Comment:
i think i did it because it felt expedient at the time, when i was in the middle of shuffling a few things around.  couldn't come up with a better reason when i thought back on it.  also wasn't wild about the way it made references in other places longer because of the prefixed containing class name.  so i broke it out into its own class file.